### ISSUE TITLE
ValueAxis percentage formatter not respecting precision

### ISSUE THREAD

**Issue by justinhelmer** [amcharts/amcharts4#1255]
When using the percentage formatter, The value axis is not rounding to the appropriate decimal place before formatting.

The ValueAxis (y) looks something like this:

<img width="454" alt="Screen Shot 2019-04-29 at 7 16 15 PM" src="https://user-images.githubusercontent.com/1403438/56937751-4d4b9500-6ab3-11e9-8113-29c408795ef6.png">

Codepen for above: https://codepen.io/anon/pen/bJJOvN

By commenting out the formatter, the precision is not an issue. For example: 

<img width="256" alt="Screen Shot 2019-04-29 at 7 17 47 PM" src="https://user-images.githubusercontent.com/1403438/56937790-7e2bca00-6ab3-11e9-968f-e2cceaa8beff.png">

Codepen for above: https://codepen.io/anon/pen/rbboKz

However, this is not ideal, as I am trying to use the percentage formatter as opposed to manually converting the values with JavaScript.

We will be rounding to the nearest integer, so we are using the `(dot)` formatter. However due to the value not being exactly 0, it results in an ugly "-0%" at the zero line, as opposed to the value "0" that is being specified in the formatter:

<img width="250" alt="Screen Shot 2019-04-29 at 7 21 03 PM" src="https://user-images.githubusercontent.com/1403438/56937892-f3979a80-6ab3-11e9-8ab3-167e65a25014.png">

Codepen for above: https://codepen.io/anon/pen/OGGrwL

We tried using `improvementAxis.maxPrecision = 1` but it had no effect.

**Comment by martynasma** [comment]
You do not seem to have any decimal rounding set in your number format:

```
improvementAxis.numberFormatter.numberFormat = '#,###%|-#,###%s|0';
```

There is no dot in there, so formatter assumes you don't need any decimal rounding.

If you'd like to round to integers, you could do the following:

```
improvementAxis.numberFormatter.numberFormat = '#,###.%|-#,###.%s|0';
```

Now, "negative zero" might be a bug. We'll be looking into it and will fix as necessary.

Thanks for bringing it to our attention!

**Comment by justinhelmer** [comment]
Thanks for getting back on this. There are three codepens there, one for each screenshot. For the screenshot that shows the "-0%", you will see I am using the dot formatter exactly as you have suggested - I think you just missed it.

Happy to bring this to your attention! Let me know if you need any more info.

**Comment by martynasma** [comment]
Yeah, that's a bug which we already fixed in dev. I think we'll release it publicly by EOW.

**Comment by martynasma** [comment]
Fixed in [4.4.2](https://github.com/amcharts/amcharts4/releases/tag/4.4.2).

## [4.4.2] - 2019-05-02

### Added
- New property for `Label`: `baseLineRatio` (default `-0.27`) controls how "base line" for a label is calculated, and affects positioning of text. The default might not work for all fonts, so this setting can be used to adjust it for precise positioning.
- New event for `Sprite`: `"dragged"`. It kicks in right after `"drag"` and can be used to manipulate or correct element's position, which might not be possible using `"drag"` which has built-in position-altering handlers.

### Fixed
- Export to SVG was not working on Firefox.
- Some touch issues wit iOS Safari fixed.
- `ValueAxis` could display non-rounded numbers (caused by floating point problem) sometimes.
- `maxPanOut` of `MapChart` was being ignored.
- Chart scrollbar was not being zoomed out when data of a chart changed.
- `RadarChart` with multiple `RadarColumnSeries` with Y as base axis was displaying columns of a 2nd+ series from wrong position.
- `chart.dispose()` with a lot of data was very slow. Now it's super fast.
- Zoom with mouse wheel was buggy when zooming already zoomed-in `XYChart`.
- Gant chart was not updating columns when `chart.invalidateRawData()` was called.
- Issue with some `propertyFields` set on a `LineSeries` with no data fixed.
- Reseting `relativeRotation` of `AxisLabelCircular` to `undefined` didn't work.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by justinhelmer** [comment]
Wow, impressive turnaround. Thanks a million. Will keep an eye on the publication of this release.
